Xamarin.Forms.Build.Tasks/XamlTask.cs

```diff
@@ -22,8 +22,7 @@ public abstract class XamlTask : MarshalByRefObject, ITask
 	{
 		[Required]
 		public string Assembly { get; set; }
-		public string DependencyPaths { get; set; }
```

Comment:
this hasn't been used in years. it's a breaking change to remove it, but unlikely to fire back.